misk-redis/src/main/kotlin/misk/redis/RealRedis.kt

```diff
@@ -302,6 +303,51 @@ class RealRedis(
     jedisPool.resource.use { jedis -> jedis.publish(channel, message) }
   }
 
+  override fun zadd(key: String, score: Double, member: ByteString, options: Set<String>): Long {
```

Comment:
should `options` be a `Set<ZAddOptions>` ? You are already exposing the `ZAddOptions` unless you meant to make that private?